plan/resolve_indices.go

```diff
@@ -216,9 +216,11 @@ func (p *Insert) ResolveIndices() {
 
 // ResolveIndices implements Plan interface.
 func (p *basePlan) ResolveIndices() {
-	for _, cols := range p.schema.TblID2Handle {
-		for _, col := range cols {
-			col.ResolveIndices(p.schema)
+	if p.schema != nil {
```

Comment:
comment for when this check will be true.

Reply:
I also don't know when it would fail. It's strange and I will refactor the schema setting in the future.